.github/workflows/pr-title-linter-and-linker.yaml

```diff
@@ -0,0 +1,82 @@
+name: "Lint and Link PR title"
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+
+jobs:
+  lint:
+    name: Validate PR title And link Jira Issue
+    runs-on: ubuntu-22.04
+    env:
+      JIRA_COMMENT_REGEX: "^.*Jira.*"
+    steps:
+      - uses: Slashgear/action-check-pr-title@860e8dc639f8e60335a6f5e8936ba67ed2536890
```

Comment:
Why we are using the commit id instead the tag to refear the github action?

Reply:
@AleDore said it was a security direction.